ps2/hangman.py

```diff
@@ -63,9 +60,9 @@ def is_word_guessed(secret_word, letters_guessed):
     letters_guessed; False otherwise
     '''
     secret_word_split = list(secret_word)
-    for letter in secret_word_split:
+    for letter in secret_word:
         if letter in letters_guessed:
-            pass
+            continue
```

Comment:
why not just 

```python
if letter not in letters_guessed:
  return False
```

Reply:
ok but then how would you handle the True condition.

Reply:
It's handled already.

Reply:
I see, this is a simpler way.